core/src/main/scala/kafka/admin/AdminClient.scala

```diff
@@ -209,6 +210,8 @@ object AdminClient {
     create(new AdminConfig(config))
   }
 
+  def create(props: Properties): AdminClient = create(propertiesAsScalaMap(props).map(x => x._1 -> x._2).toMap)
```

Comment:
Can you not simply not do (assuming import JavaConverters._): `create(props.asScala.toMap)`?